modules/trusted-firmware-m/tfm_boards/CMakeLists.txt

```diff
@@ -207,7 +207,7 @@ file(COPY ${CMAKE_CURRENT_LIST_DIR}/common
      DESTINATION ${INSTALL_PLATFORM_NS_DIR})
 
 
-if((NRF_SOC_VARIANT STREQUAL nrf54l15) OR (target STREQUAL nrf54l15))
```

Comment:
Why are you not using `CONFIG_SOC_NRF54L15` ?

Reply:
Good question. I was not the one to introduce this. Here I just cleaned up the use of `target` that seemed useless. I guess `NRF_SOC_VARIANT` was used because it's defined in the TF-M CMake logic. There is likely no good reason for `CONFIG_SOC_*` to not be used instead.

Reply:
Okay so actually there is. `CONFIG_SOC_NRF54L15` is not defined in there, so it doesn't work. I'm reverting to using `NRF_SOC_VARIANT`.